prepare.backend.sh

```
@@ -0,0 +1,11 @@
+cmake -S . -B ../build.remoting-backend \
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)